test/commands/scanner/run.test.ts

```diff
@@ -782,19 +782,24 @@ describe('scanner:run', function () {
 	describe('MultiEngine', () => {
 		describe('Project: JS', () => {
 			before(() => {
-				process.chdir(path.join('test', 'code-fixtures', 'projects'))
+				process.chdir(path.join('test', 'code-fixtures', 'projects', 'app'))
 			});
 			after(() => {
-				process.chdir("../../..");
+				process.chdir("../../../..");
 			});
 			runTest
 				.stdout()
 				.stderr()
-				.command(['scanner:run', '--target', 'js', '--format', 'json'])
-				.it('JS project triggers pmd and eslint rules', ctx => {
+				.command(['scanner:run', '--target', '**/*.js,**/*.cls', '--format', 'json'])
```

Comment:
This was previously picking up pmd and eslint via js. Changed it to include apex and js instead.